### ISSUE TITLE
ValueError("Invalid value for `stored_versions`, must not be `None`") when  calling list_custom_resource_definition

### ISSUE THREAD

**Issue by jicowan** [kubernetes-client/python#679]
When I try using the list_custom_resource_definition method, I get the error: 
```
ValueError("Invalid value for `stored_versions`, must not be `None`")
```
The output of `kubectl get crds -o yaml` doesn't include a field called `stored_versions`

I'm trying to get the list of crds installed. 
Using Python 2.7 with Kubernetes 1.10.3

**Comment by micw523** [comment]
Unfortunately you ran into
#376 
python-base#52

**Comment by MirzaSikander** [comment]
Is there a workaround for this in the meantime?

**Comment by sjpotter** [comment]
Just an FYI, this bug was never fixed and will impact anyone trying to use 1.9/1.10 (which at this point might be minimal, just should be noted)
